app/javascript/app/pages/home/home-component.jsx

```diff
@@ -48,6 +49,9 @@ const Home = () => (
     <div>
       <HighlightedStories />
     </div>
+    <div>
```

Comment:
I don't think it's needed, can you check?

Reply:
otherwise looks nice 👍

Reply:
I know, thanks